project/SparkBuild.scala

```diff
@@ -342,6 +342,18 @@ object SparkBuild extends PomBuild {
       }
     },
 
+    // Copy system properties to forked JVMs so that tests know proxy settings
+    javaOptions ++= {
```

Comment:
Given the description, it sounds like a runtime error instead of a compile error. Then, `TestSettings` via `(Test / javaOptions)`?

https://github.com/apache/spark/blob/10751dc285c5c639e3343a8abc26857407522822/project/SparkBuild.scala#L1563-L1583

Reply:
I've moved the update into TestSettings and switched to "Test / javaOptions".  I've verified these changes in my local environment.